### PR TITLE
implement getPseudorandomIndices

### PR DESCRIPTION
**Pull request by mathetake**
ref: https://github.com/ethereum/research/blob/master/mimc_stark/utils.py#L12-L23

### CODE REVIEW

File: contracts/BytesLib.sol [hunk 1]
```diff
@@ -287,11 +287,11 @@ library BytesLib {
     }
 
     function toUint(bytes _bytes, uint _start) internal  pure returns (uint256) {
-        require(_bytes.length >= (_start + 32));
+        require(_bytes.length >= (_start + 4));
         uint256 tempUint;
 
         assembly {
-            tempUint := mload(add(add(_bytes, 0x20), _start))
+            tempUint := mload(add(add(_bytes, 0x04), _start))
```
Comment (osuketh) [inline contracts/BytesLib.sol:294]: why original implementation is `0x20` ??

Comment (mathetake) [reply contracts/BytesLib.sol:294]: 0x20 = 32..........

Comment (osuketh) [reply contracts/BytesLib.sol:294]: ya!w

File: contracts/Verifier.sol [hunk 2]
```diff
@@ -154,8 +154,30 @@ contract VerifierContract {
...
+        uint[] memory out = new uint[](data.length / 4);
+        uint start = 0;
+        uint j = 0;
+        if (_excludeMultiplesOf == 0) {
+            for (j = 0; j < out.length; j++) {
+                out[j] = (data.slice(start, 4).toUint(0)).mod(_modulus);
+                start += 4;
+            }
+        } else {
+            uint _realModulus = _modulus.mul(_excludeMultiplesOf - 1);
```
Comment (osuketh) [inline contracts/Verifier.sol]: `_realModulus` -> `realModulus`

**Review by osuketh — COMMENTED** [review]
yosage

**Pull request merged**